optimum/intel/openvino/modeling_base.py

```diff
@@ -310,6 +312,12 @@ def load_model(
                 Quantization config to apply after model is loaded.
         """
 
+        if quantization_config is not None:
+            logger.warning(
+                "Providing `quantization_config` to `load_model` is deprecated and will be removed in v1.28.0. "
```

Comment:
will it be removed in 1.27 or 1.28? Please check. I may be wrong here.

Reply:
My plan was to deprecate `trust_remote_code` from `OVQuantizationConfigBase` in 1.27 and `quantization_config` from `load_model` in 1.28, because the first one is urgent.